src/test/java/part1/exercise/StreamsExercise2.java

```diff
@@ -75,8 +75,9 @@ public void employersStuffLists() {
 
     private static PersonEmployerPair firstEmployerPersonPair(Employee employee) {
         final JobHistoryEntry jobHistoryEntry = employee.getJobHistory().stream()
-                .findFirst()
-                .get();
+                .limit(1)
+                .findAny()
+                .orElse(new JobHistoryEntry(-1, "Default", "Default"));

```

Comment:
Don't use default values. You can return `Stream<PersonEmployerPair>` from this method.